sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveCatalogs.scala

```diff
@@ -94,11 +97,21 @@ class ResolveCatalogs(val catalogManager: CatalogManager)
           s"because view support in catalog has not been implemented yet")
 
     case AlterNamespaceSetPropertiesStatement(NonSessionCatalog(catalog, nameParts), properties) =>
+      if (properties.keySet.intersect(REVERSED_PROPERTIES.asScala.toSet).nonEmpty) {
+        throw new AnalysisException(s"Cannot directly modify the reversed properties" +
```

Comment:
nit: drop `s`

Reply:
Is this change related to this pr to support `SET OWNER`?

Reply:
to prohibit changing ownership ·SET PROPERTIES·